models/applicationSubmissionsModel.js

```diff
@@ -2,22 +2,23 @@ const db = require('../database/dbConfig');
 
 async function findById(id) {
   return db('application_submissions')
+    .select('*')
     .where({ id })
     .first();
 }
 
 async function findAllByCampaignId(campaign_id) {
   return db('skilled_impact_requests')
-    .where({ campaign_id })
     .join(
       'application_submissions',
       'skilled_impact_requests.id',
-      'application_submissions.skilled_impact_request_id',
+      'application_submissions.skilled_impact_request_id'
```

Comment:
I think our ESLint rules require last element commas, can you please lint this file?